ts/components/messages/MessageDueDateBar.tsx

```diff
@@ -247,8 +247,8 @@ const MessageDueDateBar: React.FunctionComponent<Props> = ({
   ));
 };
 
-const mapDispatchToProps = (dispatch: Dispatch) => ({
-  onGoToWallet: () => dispatch(navigateToWalletHome())
+const mapDispatchToProps = (_: Dispatch) => ({
+  onGoToWallet: () => navigateToWalletHome()
```

Comment:
Similar to what I wrote above. And even here, we have `connect` just to support a bogus `mapDispatchToProps`.